src/js/control-bar/time-controls/time-display.js

```diff
@@ -63,7 +63,7 @@ class TimeDisplay extends Component {
    * @private
    */
   updateTextNode_() {
-    if (this.textNode_) {
+    if (this.textNode_ && this.contentEl_ && this.contentEl_.contains(this.textNode_)) {
       this.contentEl_.removeChild(this.textNode_);
```

Comment:
IE11 throws in request animation frames when trying to remove a DOM node from a container when the container does not have the node. Might only be an issue for our unit tests, but I fixed it here.